package.json

```diff
@@ -2,6 +2,14 @@
   "name": "the-infi-way",
   "repository": "github:infi-nl/the-infi-way",
   "devDependencies": {
-    "@netlify/plugin-lighthouse": "^4.0.7"
+    "@netlify/plugin-lighthouse": "^4.0.7",
+    "http-server": "^14.1.1",
+    "nodemon": "^2.0.22"
+  },
+  "scripts": {
+    "build": "node build.js",
+    "build:watch": "nodemon -L -w template.html -w content build.js",
+    "start": "concurrently 'npm run build:watch' 'npm run serve'",
```

Comment:
You probably need to add [concurrently](https://www.npmjs.com/package/concurrently) as a dev dependency, right?

Reply:
Done now :)